modin/pandas/test/test_io.py

```diff
@@ -2056,6 +2132,10 @@ def test_read_pickle(self):
         finally:
             teardown_test_files([unique_filename])
 
+    @pytest.mark.xfail(
+        condition="config.getoption('--simulate-cloud').lower() != 'off'",
+        reason="There is no point in reading from local files.",
```

Comment:
I'm not sure if it's correct to read/write to local files remotely? It may be worth creating a separate problem to skip tests where reading/writing to local files is carried out in the `"modin in the cloud"` mode. And in this problem, correct the reason and replace `xfail` with `skip`.

Reply:
> I'm not sure if it's correct to read/write to local files remotely?

We read/write to local files for `ExperimentalPandasOnCloudpython` backend.

Reply:
Good. Then I will correct for what you suggested.

Reply:
Actually, I have this test passed locally. Why did you disable it?

Reply:
Discussed offline, decided to leave as it is because the test fails when running the whole test_io.py.